src/MassTransit/SystemTextJsonMessageSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace MassTransit.Serialization
+{
+    using MassTransit.Metadata;
+    using System;
+    using System.IO;
+    using System.Net.Mime;
+    using System.Runtime.Serialization;
+    using System.Text.Json;
+
+    public class SystemTextJsonMessageSerializer : IMessageSerializer
+    {
+        public ContentType ContentType => JsonContentType;
+
+        internal static readonly ContentType JsonContentType = new ContentType("application/vnd.masstransit+json2");
```

Comment:
json2? I wish I could think of something better than `2` here...

Reply:
`system-json`, `text-json`?

Reply:
`application/vnd.masstransit+text-json` if it's legal would be alright.

Reply:
done.